settings.py

```diff
@@ -54,7 +55,20 @@ def update(setting_name, updated_value):
     update_settings_file(settings)
 
 def setup_logger():
+    handlers = []
+    fh = logging.FileHandler(LOG_FILE_PATH)
```

Comment:
Nit: No reason to use abbreviations here.

fileHandler = logging.FileHandler(LOG_FILE_PATH) is better!